app/serializers/license_serializer.rb

```diff
@@ -28,6 +28,9 @@ class LicenseSerializer < BaseSerializer
   attribute :protected do
     @object.protected?
   end
+  attribute :version do
+    @object.last_validated_version
```

Comment:
Would be cool to track this over time, and not just the version for the most recent scoped validation.

But we can do that later.